kolibri_instant_schools_plugin/assets/src/views/SignUpPage.vue

```diff
@@ -110,18 +110,13 @@
         v-if="showTerms"
         :title="$tr('termsOfServiceModalHeader')"
         :size="'large'"
+        :cancelText="$tr('close')"
         @cancel="showTerms = false"
       >
         <iframe
           class="terms"
-          src="/content/databases/about/tos.txt"
+          src="/en/user/api/aboutfaq/Terms"
```

Comment:
Should the language be hardcoded here like this? Seems like this might want to change depending on the UI language?

Reply:
That endpoint just returns whatever AboutFAQ record that is of the 'terms' kind so I can set/update it in the Django admin